src/main/java/org/gradle/profiler/GradleBuildConfiguration.java

```diff
@@ -13,13 +14,38 @@
     private final File javaHome;
     private final List<String> jvmArguments;
     private final boolean usesScanPlugin;
+    /**
+     * The GRADLE_OPTS environment variable passed to configure the client VM.
+     * See https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables
+     * <p>
+     * This is used only for --daemon. Use jvmArguments for --no-daemon.
+     */
+    private final List<String> gradleOpts;
```

Comment:
Let's call this `clientJvmArgs`. I think the Javadoc should go to the getter. We should make it clear here that:
- This is only for the CLI invoker,
- and only when the daemon is used.

It would be great to only have this on the CLI-invoker, though I don't see a good way how to do so.

Reply:
Yeah... my first attempt also goes into Cli invoker only but failed...